include/psen_scan_v2/diagnostics.h

```diff
@@ -137,14 +141,15 @@ class ErrorLocation
   BitLocation bit_;
 };
 
-class MonitoringFrameDiagnosticMessage
+class Message
 {
 public:
-  constexpr MonitoringFrameDiagnosticMessage(const ScannerId& id, const ErrorLocation& location);
+  constexpr Message(const ScannerId& id, const ErrorLocation& location);
 
-  constexpr bool operator==(const MonitoringFrameDiagnosticMessage& rhs) const;
+  constexpr bool operator==(const Message& rhs) const;
 
-  friend RawDiagnosticMsg serializeDiagnosticMessages(const std::vector<MonitoringFrameDiagnosticMessage>& messages);
+  friend diagnostic::raw_message::Type
+  psen_scan_v2::monitoring_frame::diagnostic::serializeMessages(const std::vector<diagnostic::Message>& messages);
```

Comment:
No need to state the obvious. The code should state only the namespace part increasing readability.
```suggestion
  diagnostic::serialize(const std::vector<diagnostic::Message>& messages);
```